scripts/globals/wsquest.lua

```diff
@@ -20,6 +20,7 @@ dsp.wsquest =
         ['logId'] = BASTOK,
         ['questId'] = THE_WALLS_OF_YOUR_MIND,
         ['fameRegion'] = BASTOK,
+        ['wsnmName'] = 'Bodach',
```

Comment:
Why are these needed at all?

You can only have one trial active at a time, so it's not like you'll have a collision. Just use a generic `killed_wsnm` variable name or something and do away with getvariablename/wsnmName all together imo.